packages/frontend/pages/profile/[addr].jsx

```diff
@@ -14,7 +14,7 @@ const ProfilePage = ({ user }) => {
 
 export async function getServerSideProps(context) {
   const { addr } = context.params;
-  const user = await fetch(`http://localhost:3000/api/lukso/l16/${addr}`);
+  const user = await fetch(`http://evm.pinsave.app/api/lukso/l14/${addr}`);
```

Comment:
nice idea to link it to the Prod BE, but I think we'll have to swap this to Environment variables so we can access Dev endpoints while running locally (will push some tweaks regarding that)